Common.php

```diff
@@ -134,12 +134,12 @@ function _getTabs()
      * @return   string
      * @access   private
      */
-    function _getAttrString($attributes)
+    protected function _getAttrString(array $attributes)
     {
         $strAttr = '';
 
         if (is_array($attributes)) {
-            $charset = HTML_Common::charset();
+            $charset = $this->charset();
```

Comment:
Should use self::charset() instead because html_common is used to know which charset is used globally.

Reply:
Updated in https://github.com/pear/HTML_Common/pull/3/commits/d5b198ce0da7f6923e1c918039abb6301f6bcaff